homeassistant/components/screenlogic/__init__.py

```diff
@@ -56,14 +56,14 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         hass, config_entry=entry, gateway=gateway
     )
 
-    async_load_screenlogic_services(hass)
-
     await coordinator.async_config_entry_first_refresh()
 
     entry.async_on_unload(entry.add_update_listener(async_update_listener))
 
     hass.data.setdefault(DOMAIN, {})[entry.entry_id] = coordinator
 
+    async_load_screenlogic_services(hass, entry)
```

Comment:
Integration services should nowadays be registered in `async_setup` regardless of config entry state.

See example in fully_kiosk integration.

See this conversation for the reason:
https://github.com/home-assistant/core/pull/105929#discussion_r1433754113